src/Eagle.ts

```diff
@@ -4268,12 +4263,12 @@ export class Eagle {
 
         setTimeout(function() {
             //handling selecting and highlighting the newly created node
-            const clickTarget = $($("#paramsTableWrapper tbody").children()[fieldIndex]).find('.selectionTargets')[0]
+            const clickTarget = $($(".paramsTableWrapper tbody").children()[fieldIndex]).find('.selectionTargets')[0]
```

Comment:
Also, it may be possible to do this with one CSS selector. Something like:

`const clickTarget = $(".paramsTableWrapper tbody:nth-child(" + fieldIndex + ") .selectionTargets")[0]`

Reply:
If it is not possible, or difficult, then don't worry about it. It is just the double $() that spooked me.

Reply:
the outer $() in this case is just to turn the return object of whats inside into a jquery object, so we can use .find(). 
ill give your method a try

Reply:
i made it work with this line:
            const clickTarget = $(".paramsTableWrapper tr:nth-child(" + (fieldIndex+1) + ") .selectionTargets")[0]

.children() returns a list of whats inside of the given object. so from your line i had to change "tbody:nth-child" to "tr:nth-child". and i guess .children starts counting at 0 while nth-child starts counting at 1, because i had to add 1 to get to the same item in the list. 
note, fieldIndex is used again after this, so i cant increase the field index before this line